### PR TITLE
Feature/delete button

### LINKED ISSUE #10: user story 6 - delete button
## Value Proposition
As a user, I want to be able to delete an entry, so that I might correct any mistakes I made.

## Description

Screenshot 2022-10-19 at 12 37 38

## Acception Criteria
- [x] display a delete button on each card
- [x] delete the card, when clicking the button
- [x] delete the entry from the data base

## Tasks
- [x] create delete button
- [x] display delete button
- [x] get ID from clicked entry
- [x] search for id in data base
- [x] delete entry with id in data base

### PR DESCRIPTION
**Pull request by Mark-Krensel**
Please check the delete function code on the main page.

### CODE REVIEW

File: components/Card.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import styled from "styled-components";
 
-export default function Card({ date, weight, height }) {
+export default function Card({ date, weight, height, handleDelete, id }) {
   return (
     <CardElement>
+      <DeleteButton onClick={() => handleDelete(id)}>X</DeleteButton>
```
Comment (CathrinTruchan) [inline components/Card.js]: The button should have an accessable text (e.g. aria-label or button-text)

Comment (Mark-Krensel) [reply components/Card.js]: this is very true!

Comment (Mark-Krensel) [reply components/Card.js]: gave it an aria-label and a button type :)

**Review by ThomasF85 — APPROVED** [review]
Looks good!

**Pull request merged**